app/assets/javascripts/preventMultipleFormSubmissions.js

```diff
@@ -0,0 +1,15 @@
+(() => {
+    let frm = document.querySelector('form')
```

Comment:
No need to change it if you do not want, but as a general rule I would use `const` instead of `let`